app/code/local/Mage/Core/Model/Session/Abstract/Varien.php

```diff
@@ -116,7 +116,21 @@ public function start($sessionName=null)
             $cookieParams['domain'] = $cookie->getDomain();
         }
 
-        call_user_func_array('session_set_cookie_params', $cookieParams);
+        if (Mage::getStoreConfig(Mage_Core_Model_Cookie::XML_PATH_COOKIE_SAME_SITE)) {
```

Comment:
This constant is not available in Magento 1.9.4.5, right? This leads to an issue then.